core/src/main/java/org/springframework/security/oauth2/server/authorization/jose/KeyManager.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.server.authorization.jose;
+
+import java.security.Key;
+import java.util.List;
+
+/**
+ * A manager for the {@link Key}s.
+ *
+ * @author Anoop Garlapati
+ * @since 0.0.1
+ */
+public interface KeyManager {
```

Comment:
I wonder if this should be called `KeyService`. Several other interfaces that retrieve resources in Spring Security are services, like `UserDetailsService`, `OAuth2UserService`, `AclService`, etc.